session/tidb.go

```diff
@@ -118,6 +118,19 @@ var (
 	statsLease = int64(3 * time.Second)
 )
 
+func setStoreBootstrap(storeUUID string) {
```

Comment:
These codes are best not to be placed here. Maybe we can put it to util/testutil that dedicates to testing.

Reply:
function `finishBootstrap` in also call this to set the flag